projects/miopen/src/kernels/MIOpenCol2Im2d.cpp

```diff
@@ -24,6 +24,7 @@
  *
  *******************************************************************************/
 #include "float_types.h"
+#include <cstdint>
```

Comment:
```suggestion
#include "miopen_cstdint.hpp"
```
https://github.com/ROCm/rocm-libraries/blob/develop/projects/miopen/src/kernels/miopen_cstdint.hpp